.buildkite/pr.yml

```diff
@@ -0,0 +1,47 @@
+tapendium-aliases:
+  lib-common: &lib-common
+    - seek-oss/github-merged-pr#v1.1.2:
+        mode: checkout
+    - tapendium/1password-secrets#v2.3.0:
+        file:
+          - path: .npmrc.tpl
+            out: .npmrc
+    - nienbo/cache#v2.4.15:
+        id: testcontainers-dynamodb
+        key: >
+          "v3-cache-{{ id }}-{{ runner.os }}-{{ checksum 'package-lock.json' }}"
+        backend: s3
+        s3:
+          bucket: tap-buildkite-cache
+        paths:
+          - node_modules
+    - tapendium/asdf#v1.0.0: ~
+
+steps:
+  - group: ':lint-roller: Lint'
+    steps:
+      - label: ':yaml: YAML Lint'
+        plugins:
+          - seek-oss/github-merged-pr#v1.1.2:
+              mode: checkout
+          - docker#v5.7.0:
+              command: ['.']
+              image: 'cytopia/yamllint:latest'
+
+      - label: ':npm: npm install'
+        commands: npm ci
```

Comment:
**suggestion:** not a big one but if you used the tap-ci utils plugin this step might be quicker

Reply:
good suggestion! gonna use that [6102fbc](https://github.com/tapendium/testcontainers-dynamodb/pull/1/commits/6102fbc84c886a5a87a76da0f1d07bf95bce874a)